va/va.c

```diff
@@ -2286,3 +2299,135 @@ vaQueryVideoProcPipelineCaps(
     VA_TRACE_RET(dpy, status);
     return status;
 }
+
+VAStatus
+vaCopy(
```

Comment:
this is your change?

Reply:
No, maybe I should create a new one.
This is rebased to intel:master.

Reply:
corrected this.